test/json_samples.jl

```diff
@@ -296,8 +296,7 @@ twitter = "{\"results\":[
      \"next_page\":\"?page=2&max_id=1480307926&q=%40twitterapi\",
      \"completed_in\":0.031704,
      \"page\":1,
-     \"query\":\"%40twitterapi\"}
-}"
```

Comment:
There was an extra close brace in the original test, so it was invalid JSON. This change makes `parse` more picky about that. I don't think it's a good idea to silently accept extra characters after the end.